Dockerfile

```diff
@@ -1,4 +1,7 @@
-FROM        sdurrheimer/alpine-golang-make-onbuild
+FROM        prom/busybox:glibc
 MAINTAINER  The Prometheus Authors <prometheus-developers@googlegroups.com>
 
-EXPOSE      9100
+COPY node_exporter  /bin/node_exporter
+
+EXPOSE     9100
+ENTRYPOINT [ "/bin/node_exporter" ]
```

Comment:
So concretely, no, let's specify a sane CMD here but otherwise let users
call `node_exporter` themselves. `go-run ` can be made obsolete by copying
the binary to `/bin/go` instead of a dynamic path.